app/controllers/dashboard_controller.rb

```diff
@@ -2,8 +2,117 @@
 
 class DashboardController < ApplicationController
   before_action :verify_current_user
+  before_action :init_graphql_client
 
   def index
+    job_results = @smart_village.query <<~GRAPHQL
+      query {
+        genericItems(genericType: "Job") {
+          id
+          title
+          payload
+          dataProvider {
+            name
+          }
+          updatedAt
+          createdAt
+        }
+      }
+    GRAPHQL
+
+    news_results = @smart_village.query <<~GRAPHQL
+      query {
+        newsItems {
+          id
+          title
+          dataProvider {
+            name
+          }
+          visible
+          contentBlocks {
+            title
+          }
+          updatedAt
+          createdAt
+          pushNotificationsSentAt
+        }
+      }
+    GRAPHQL
+
+    event_results = @smart_village.query <<~GRAPHQL
+      query {
+        eventRecords {
+          id
+          title
+          visible
+          dataProvider {
+            name
+          }
+          dates {
+            dateStart
+            dateEnd
+          }
+          updatedAt
+          createdAt
+        }
+      }
+    GRAPHQL
+
+    offer_results = @smart_village.query <<~GRAPHQL
+      query {
+        genericItems(genericType: "Offer") {
+          id
+          title
+          dataProvider {
+            name
+          }
+          updatedAt
+          createdAt
+        }
+      }
+    GRAPHQL
+
+    poi_results = @smart_village.query <<~GRAPHQL
+      query {
+        pointsOfInterest {
+          id
+          name
+          visible
+          dataProvider{
+            name
+          }
+          updatedAt
+          createdAt
+        }
+      }
+    GRAPHQL
+
+    tour_results = @smart_village.query <<~GRAPHQL
+      query {
+        tours {
+          id
+          name
+          visible
+          dataProvider {
+            name
+          }
+          updatedAt
+          createdAt
+        }
+      }
+    GRAPHQL
```

Comment:
ich glaube heir könnten wir auf einige werte verzichten, die im dashboard gar nicht angezeigt werden. wir brauchen dort lediglich die anzahl an elementen. somit würde es meines erachtens nach reichen, nur die `id` in den queries aufzulisten. dies entlastet den server bei der anfrage und resultiert evtl auch in kleinem performance boost beim laden der seite, da weniger operationen ausgeführt werden müssen zum returnen der angeforderten daten.

Reply:
das stimmt, ich habe alle außer id rausgenommen und es funktioniert weiterhin alles.